pytorch_blade/benchmark/torch-tensorrt/README.md

```diff
@@ -0,0 +1,130 @@
+# Performance Benchmarking
+
+**NOTE that this package is modified from**\
+[Torch-TensorRT/exmaples/benchmark/py/README.md](https://github.com/NVIDIA/Torch-TensorRT/blob/95d2b6e003a0392fe08ce49520f015c230d4c750/examples/benchmark/py/README.md)
+
+This is a comprehensive Python benchmark suite to run perf runs using different
+supported backends. The following backends are supported:
+
+- Torch
+- Torch-TensorRT
+- TensorRT
+- Torch-Blade
+
+Note: Please note that for ONNX models, the user can convert the ONNX model to
+TensorRT serialized engine and then use this package.
+
+## Prerequisite
+
+Benchmark scripts depend on the following Python packages in addition to
+[requirements.txt](../requirements.txt) packages
+
+- Torch-TensorRT
+- Torch
+- TensorRT
+- Torch-Blade
+
+## Structure
+
+```text
+./
+├── config
+│   ├── crnn.yml
+│   ├── vgg16.yml
+│   └── yolov5.yml
+├── models
+├── perf_run.py
+├── test_trt_benchmark.sh
+└── README.md
+```
+
+Please save your configuration files at config directory. Similarly, place your
+model files at models path.
+
+## Usage
+
+To run the benchmark for a given configuration file:
+
+```shell
+python perf_run.py --config=config/vgg16.yml
+```
+
+## Configuration
+
+There are two sample configuration files added.
+
+- crnn.yml, vgg16.yml, and yolov5.yml demonstrates a configuration with all the
+  supported backends (Torch, Torch-TensorRT, TensorRT, Torch-Blade)
+
+### Supported fields
+
+| Name | Supported Values | Description |
+| --- | --- | --- |
+| backend | all, torch, torch_tensorrt, tensorrt | Supported backends for inference. |
+| input | - | Input binding names. Expected to list shapes of each input bindings |
+| model | - | Configure the model filename and name |
+| filename | - | Model file name to load from disk. |
+| name | - | Model name |
+| runtime | - | Runtime configurations |
+| device | 0 | Target device ID to run inference. Range depends on available GPUs |
```

Comment:
so, `1` is not a supported device value?

Reply:
> "Range depends on available GPUs"

The readme is modified from Torch-TensorRT's.